### PR TITLE
issue-2513: Progress Indicator: fixed optionalText styling to make it consistent with react carbon component

### PR DESCRIPTION
**Pull request by rajeshkumaru**
Closes carbon-design-system/carbon-components-angular#2513

#### Changelog

**Before**

Screenshot 2023-04-07 at 2 07 40 PM

**Changed**

Vertical progress indicator:

Screenshot 2023-04-07 at 2 07 59 PM

Horizontal progress indicator:
Screenshot 2023-04-07 at 2 53 07 PM

### CODE REVIEW

File: src/progress-indicator/progress-indicator.component.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ import { Step } from "./progress-indicator-step.interface";
 			class="bx--progress-step bx--progress-step--{{step.state[0]}}"
 			*ngFor="let step of steps; let i = index"
 			[ngClass]="{'bx--progress-step--disabled' : step.disabled}">
-			<div class="bx--progress-step-button bx--progress-step-button--unclickable" role="button" tabindex="-1">
+			<button class="bx--progress-step-button bx--progress-step-button--unclickable" role="button" tabindex="-1">
```
Comment (rajeshkumaru) [inline src/progress-indicator/progress-indicator.component.ts:31]: Converted to `button` to fix gap btw text and optionalText. (react has button)
Screenshot 2023-04-07 at 2 51 54 PM

**Review by rajeshkumaru — COMMENTED** [review]
I have read the DCO document and I hereby sign the DCO.

**Review by rajeshkumaru — COMMENTED** [review]
recheck

**Comment by Akshat55** [comment]
@rajeshkumaru Thanks for the PR! DCO is failing because you've used your IBM Github to user.name/user.email I believe. Could you please close this PR & remake it where you make the github commit when global/project user.email/user.name is set to your public github instead of IBM github?

**Comment by rajeshkumaru** [comment]
> @rajeshkumaru Thanks for the PR! DCO is failing because you've used your IBM Github to user.name/user.email I believe. Could you please close this PR & remake it where you make the github commit when global/project user.email/user.name is set to your public github instead of IBM github?

@Akshat55 , created new PR - https://github.com/carbon-design-system/carbon-components-angular/pull/2515

Thank you!

**Pull request closed without merging**